include/itkBlockMatchingMultiResolutionIterationObserver.hxx

```diff
@@ -48,7 +48,7 @@ MultiResolutionIterationObserver<TMultiResolutionMethod>::MultiResolutionIterati
 
 
 template <typename TMultiResolutionMethod>
-MultiResolutionIterationObserver<TMultiResolutionMethod>::~MultiResolutionIterationObserver()
+MultiResolutionIterationObserver<TMultiResolutionMethod>::~MultiResolutionIterationObserver() override
```

Comment:
From [CDash](https://open.cdash.org/viewBuildError.php?buildid=7665861):

```
include/itkBlockMatchingMultiResolutionIterationObserver.hxx:51:95: error: virt-specifiers in 'MultiResolutionIterationObserver' not allowed outside a class definition
```

I don't think the `override` is need in the implementation here if it is included in the declaration in `itkBlockMatchingMultiResolutionIterationObserver.h `